otherdave/util/madlib.py

```diff
@@ -8,14 +8,16 @@
 class MadLibber():
     def make(self):
         template = self.actions["template"]()
+        return self._format_template(template)
+
+    def _format_template(self, template):
```

Comment:
The reason it's split is not just for validation purposes but because templates can be grabbed in different ways because of the whole variant stuff with horoscope.

Reply:
Couldn't you just use a default parameter, like:

```
def make(self, template = None):
    if(template is None):
        template = self.actions["template"]()
    
    ... rest of code that was moved to _format_template(self, template) ...
```

Reply:
@ibanner56 will go in and address once this is checked in.